raster-foundry/raster-foundry
# Replace Airflow with AWS Batch Calls in API Server

## Overview

This commit removes support for making Airflow calls from the API server based
on user activity.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Notes

Also fixes an issue where errors were not being logged properly in the akka server's http handler.

## Testing Instructions

 * Ensure `.env` file is up to date
 * Start API server and front end (sometimes it helps to compile the API server first)
 * Upload a file to kick off an upload and ensure one gets started in AWS Batch's staging environment
 * Add a Landsat 8 scene to a project and confirm an ingest gets kicked off
 * Perform an export (do not draw a polygon though) and confirm an export gets kicked off

All AWS batch jobs will fail because the staging environment cannot communicate with the local dev environment; however, the above will confirm that jobs will be kicked off appropriately.

Closes #2506 
Closes #2485
